flow/layers/backdrop_filter_layer.cc

```diff
@@ -14,6 +14,9 @@ void BackdropFilterLayer::Preroll(PrerollContext* context,
   Layer::AutoPrerollSaveLayerState save =
       Layer::AutoPrerollSaveLayerState::Create(context, true, bool(filter_));
   ContainerLayer::Preroll(context, matrix);
+
+  // paint bounds is determined by clip, not children
+  set_paint_bounds(context->cull_rect);
```

Comment:
Other layers with custom paint bounds will use PrerollChildren rather than Preroll. The Preroll call is just a helper method that does a set_paint_bounds to the union of the children which we are going to ignore.

Reply:
Also, the top level cull_rect is kGiantRect which makes this layer and all that it inherits from have a huge bounds unless there is a clip widget above it.